### PR TITLE
Update to 21.04

### PR DESCRIPTION
**Pull request by stephdl**
Time to update to 21.04, it seems there is no breaking changes this time

### CODE REVIEW

**Comment by jookk** [comment]
Upgrade successful, working fine

File: nethserver-ejabberd.spec [hunk 1]
```diff
@@ -90,7 +90,7 @@ mkdir -p %{buildroot}/%{_localstatedir}/lib/nethserver/ejabberd/upload
 - Remove TLS 1.0 and TLS 1.1 - NethServer/dev#6170
 
 * Tue Apr 28 2020 Giacomo Sanchietti <giacomo.sanchietti@nethesis.it> - 1.7.0-1
-- Ejabberd: upgrade to 20.03 - NethServer/dev#6108
+- Ejabberd: upgrade to 21.04 - NethServer/dev#6108
```
Comment (gsanchietti) [inline nethserver-ejabberd.spec:93]: Please revert this change.

**Review by gsanchietti — CHANGES_REQUESTED** [review]
Did you test BOSH integration with WebTop?

**Comment by stephdl** [comment]
> Did you test BOSH integration with WebTop?

@gsanchietti i did not test it, what is supposed to do ?

**Comment by gsanchietti** [comment]
You can create 2 webtop users and see if they can exchange messages from within webtop itself. See https://docs.nethserver.org/en/v7/webtop5.html#chat-integration

**Comment by stephdl** [comment]
In the meanwhile I was looking to https://docs.nethserver.org/en/v7/webtop5.html#audio-and-video-webrtc-calls-with-chat-beta

**Comment by stephdl** [comment]
The conversation with ejabberd and users in webtop is fine, the only things broken is the groupchat but this is true also with the 20.03 version

Screenshot - 2021-05-19T162254 340
Screenshot - 2021-05-19T162202 282

see [gist](https://gist.github.com/stephdl/367f9fee51d13e9e08430273131612c2) for log information (20.03 version of ejabberd)

File: nethserver-ejabberd.spec (hunk 1 above)
Comment (stephdl) [reply nethserver-ejabberd.spec]: this is done

**Comment by stephdl** [comment]
do we release it now or with a minor centos update ?

**Comment by stephdl** [comment]
Let's wait another request from the community, there is not much interest

**Pull request closed without merging**